academic-services/project-academic-knowledge/extended-attribute-deprecation-migration.md

```diff
@@ -0,0 +1,33 @@
+# Deprecation of Entity Extended Attributes
+
+The Extended Entity Attributes is being deprecated and Evaluate/Histogram API will start return attribute not found error when requested on December 21, 2020. The Extended Entity Attributes provides extended entity information such as paper BibTex venue name, citation contexts, original paper title, and DOI are being deprecated. Equivalent information can be found from other available attributes. The other equivalent attributes can also perform more operations. Please transition your API usage to leverage the equivalent available attributes.
```

Comment:
>deprecated [](start = 40, length = 10)

The attribute is already deprecated, as it is usable but is labeled as obsolete in our documentation. I believe what you're trying to say here is the attribute is being decommissioned/removed on December 21st, 2020.